firmware/src/rdm_handler.c

```diff
@@ -41,6 +43,116 @@ typedef struct {
 
 static RDMHandlerState g_rdm_handler;
 
+static int GetSetModelId(const RDMHeader *header,
```

Comment:
Not quite, this exists outside of the dispatching function, so we don't test it with function pointer comparisons.